languages/ocaml.scm

```diff
@@ -557,6 +558,7 @@
   ">" @prepend_space
 )
 
+; NOTE This rule is redundant now that spacing is added around ::
```

Comment:
I think we should remove this rule altogether, and add a test case which breaks if we remove the spacing (i.e. add `foo ~arg :: []` somewhere in the OCaml input.

Reply:
That's a good idea :+1: I just left it commented out in this draft, to make the consequences clear.

Reply:
Changed in d14e6b05b1a7f4